packages/vue-components/src/Overlay.vue

```diff
@@ -57,7 +57,12 @@ export default {
   },
   methods: {
     toggleNavMenu() {
-      if (!this.show) { publish('closeOverlay'); }
+      if (!this.show) {
+        publish('closeOverlay');
+        document.body.style.overflow = 'hidden';
+      } else {
+        document.body.style.removeProperty('overflow');
+      }
       this.show = !this.show;
```

Comment:
This extra block is needed so that when the `overlay` is opened, it prevents the header from hiding as a result of scroll chaining scrolling the body.